stdlib/tkinter/__init__.pyi

```diff
@@ -344,6 +347,7 @@ class Misc:
     def tk_focusNext(self) -> Misc | None: ...
     def tk_focusPrev(self) -> Misc | None: ...
     @overload
+    @deprecated(".after(timeout) without a callback function freezes the GUI app, use .after(timeout, func) instead")
```

Comment:
This would be a better use case for `@always_error` (or whatever the name would be) than `@deprecated`, but IMO `@deprecated` is much better than nothing here.

Reply:
This is the only one that's crashing stubtest and pytype (I think) -- you could just defer this one for now